### DISCUSSION TITLE
Smart IR - Home Assistant - Amplificatore NAD

### DISCUSSION THREAD

**Discussion by macuser79** [indomus/forum#3035] [HUB: Home Assistant]
Ciao a tutti,
ho un problema nella gestione dell'amplificatore NAD all'interno di Home Assistant.
E' stato creato il file json con tutti i codici necessari, vedi ad esempio, comando di accensione, cambio canale, aumento volume, diminuzione volume ed eseguiti tramite Broadlink manager questi funzionano correttamente.
Ho però il problema che nonostante tutti i comandi siano funzionanti, aggiungendo la card nella Dashboard,  riesco ad accendere l'amplificatore ma non riesco a spegnerlo.
Credo che il problema sia legato al fatto che SmartIR non gestisce correttamente lo stato ON dell'amplificatore, di conseguenza, non capisce che l'amplificatore è acceso e invia sempre il comando di ON anche quando il NAD è attivo.
Come posso gestire questa cosa ? Teoricamente dovrei prendere una spina che gestisce il carico della corrente ?
Grazie,
Vincenzo
[2000.json](https://github.com/indomus/forum/files/14954718/2000.json)

**Comment by ignaz72** [comment]
Anche io ho un nad e il pulsante di accensione è diverso da quello per spegnere sono comandi diversi. E funziona. Normalmente smartir non ha lo stato potresti usare un aiutante ma sarebbe uno stato fittizio se il comando non va a buon fine sarebbe comunque in on

**Comment by KC-inDomus** [comment]
> Ciao a tutti, ho un problema nella gestione dell'amplificatore NAD all'interno di Home Assistant. E' stato creato il file json con tutti i codici necessari, vedi ad esempio, comando di accensione, cambio canale, aumento volume, diminuzione volume ed eseguiti tramite Broadlink manager questi funzionano correttamente. Ho però il problema che nonostante tutti i comandi siano funzionanti, aggiungendo la card nella Dashboard, riesco ad accendere l'amplificatore ma non riesco a spegnerlo. Credo che il problema sia legato al fatto che SmartIR non gestisce correttamente lo stato ON dell'amplificatore, di conseguenza, non capisce che l'amplificatore è acceso e invia sempre il comando di ON anche quando il NAD è attivo. Come posso gestire questa cosa ? Teoricamente dovrei prendere una spina che gestisce il carico della corrente ? Grazie, Vincenzo [2000.json](https://github.com/indomus/forum/files/14954718/2000.json)

sì, implementa un misuratore di potenza e conseguentemente un binary_sensor per indicare lo stato a smartir.
